README.md

```diff
@@ -23,86 +23,116 @@
 
 [![Short demo](./logo/demo.gif)](https://youtu.be/_c4U6ffumw8)
 
+## Welcome to the Wing Language! :wave:
 
-# Welcome! :wave:
+Introducing **Wing**, an open-source, statically-typed, cloud-oriented programming language that is ready to give your cloud development a lift! 💨
```

Comment:
> give your cloud development a lift

Maybe a little too fluffy to include in the very first sentence?

Reply:
OK - I've swapped "that is ready to give your cloud development a lift" with "that puts the whole cloud at your command". Just trying stuff out 🙂 

(I feel like it's still important to hook the reader as soon as possible, even as soon as the first sentence, though hopefully this fluff is less... fluffy)

Made some more tweaks to the intro as well